tests/integration/tier0/non-destructive/kernel/test_kernel_check_verification.py

```diff
@@ -203,10 +203,7 @@ def test_non_latest_kernel_error(kernel, shell, convert2rhel):
             )
         ) as c2r:
             c2r.expect("Check if the loaded kernel version is the most recent")
-            if c2r.expect("IS_LOADED_KERNEL_LATEST:INVALID_KERNEL_VERSION") == 0:
-                c2r.sendcontrol("c")
-            else:
-                assert (
-                    AssertionError
-                ), "Utility did not raise: ERROR - (ERROR) IS_LOADED_KERNEL_LATEST.INVALID_KERNEL_VERSION"
+            c2r.expect_exact("(OVERRIDABLE) IS_LOADED_KERNEL_LATEST:INVALID_KERNEL_VERSION")
```

Comment:
non-blocking, but you can update the `test_non_latest_kernel_error()` description, it still contains the word "ERROR"